src/ui/widgets/WidgetForm.js

```diff
@@ -0,0 +1,143 @@
+            </span>
+          }
+          placeholder={formattedText('widget.page.create.code.placeholder')}
+          help={<FieldLevelHelp content={formattedText('widget.help.code')} />}
+          validate={[required, widgetCode]}
+        />
+
+        <Field
+          component={RenderTextInput}
+          name="titles.en"
+          label={
+            <span>
+              <span className="label label-info">
+                <FormattedMessage id="app.en" />
+              </span>
+              <FormattedMessage id="widget.page.create.title" />
+              <i className="fa fa-asterisk required-icon" />
+            </span>
+          }
+          placeholder={formattedText('widget.page.create.title.en.placeholder')}
+          validate={[required, maxLength(255)]}
+        />
+        <Field
+          component={RenderTextInput}
+          name="titles.it"
+          label={
+            <span>
+              <span className="label label-info">
+                <FormattedMessage id="app.it" />
+              </span>
+              <FormattedMessage id="widget.page.create.title" />
+              <i className="fa fa-asterisk required-icon" />
+            </span>
+          }
+          placeholder={formattedText('widget.page.create.title.it.placeholder')}
+          validate={[required, maxLength(255)]}
+        />
+        <div className="form-group">
+          <label htmlFor="mainGroup" className="col-sm-2 control-label">
+            <FormattedMessage id="widget.page.create.group" />
+          </label>
+          <div className="col-sm-10">
+            <Field name="group" component="select" className="form-control">
+              {fetchGroups()}
+            </Field>
+          </div>
+        </div>
+      </fieldset>
+      <fieldset className="col-xs-12 col-md-12 no-padding">
+        <div className="col-xs-12">
+          <div className="form-group">
+            <span className="control-label col-sm-2" />
+            <div className=" col-sm-10">
+              <Tabs id="basic-tabs" defaultActiveKey={1}>
+                <Tab eventKey={1} title={formattedText('widget.page.tab.customUi')} >
+                  <div className="tab-content margin-large-bottom ">
+                    <div className="tab-pane fade in active" id="widget-gui">
+                      <Field
+                        name="customUi"
+                        component="textarea"
+                        cols="50"
+                        rows="8"
+                        className="form-control"
+                        validate={[required]}
+                      />
+                    </div>
+                    <div className="tab-pane isRequiredfade" id="widget-default-gui" />
+                  </div>
+                </Tab>
+              </Tabs>
+            </div>
+          </div>
+        </div>
+      </fieldset>
+      <br />
+      <Row>
+        <div className="form-group col-md-12">
```

Comment:
Why using \<Row\> but not using \<Col\> ? also, let's avoid "md" bootstrap breakpoint in favor of "xs" (so that we don't have strange behaviors when resizing the browser). E.g., use ```<Col xs={12}>```, equivalent to "col-xs-12" class